app/components/unregistered-contributor-form.js

```diff
@@ -0,0 +1,45 @@
+import Ember from 'ember';
+
+import {
+  validator, buildValidations
+}
+from 'ember-cp-validations';
+
+/**
+* Validations for adding unregistered contributor form.  fullName must be present
+* and have three letters, and the username (email) must be present and of appropriate format.
+*/
+const Validations = buildValidations({
+    fullName: {
+        description: 'Full Name',
+        validators: [
+            validator('presence', true),
+            validator('length', {
+                min: 3
+            })
+        ]
+    },
+    username: {
+        validators: [
+            validator('presence', true),
+            validator('format', {
+                type: 'email'
+            })
+       ]
+    },
+});
+
+export default Ember.Component.extend(Validations, {
+    fullName: null,
+    username: null,
+    isFormValid: Ember.computed.alias('validations.isValid'),
+    actions: {
+        emptyView() {
```

Comment:
Why does this action need to be passed in? And is the boilerplate here really necessary?

(can you just do something like `{{action emptyView}}`, using the passed in action name instead of a string reference? The more error handling you need in this component the less viable that is, of course)